addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -946,15 +946,30 @@ var SnippetsMenu = Widget.extend({
         // again when the mouse moves
         this.$document.on('keydown.snippets_menu', () => {
             this.snippetEditors.forEach(editor => {
-                editor.toggleTextEdition(true);
+                editor.toggleOverlayHidden(true);
             });
         });
         this.$document.on('mousemove.snippets_menu, mousedown.snippets_menu', () => {
             this.snippetEditors.forEach(editor => {
-                editor.toggleTextEdition(false);
+                editor.toggleOverlayHidden(false);
```

Comment:
Does this mean that if we move during scrolling = most of the time, it does not work?

Reply:
no, it worked well like that. but I still made some changes so that the code is not run too much for nothing (throttle on mousemove + avoid to execute the code in mousemove code if it's scrolling)